virtualenv.py

```diff
@@ -354,11 +354,21 @@ def copyfile(src, dest, symlink=True):
         logger.info('Copying to %s', dest)
         copyfileordir(src, dest, symlink)
 
-def writefile(dest, content, overwrite=True):
+def get_system_encoding():
+    if is_win:
+        # .encode('mbcs') is broken on Windows. For example,
+        # u'\u00a3'.encode('mbcs') gives a wrong result in CP437 (en-US), and
```

Comment:
On other systems ```sys.getfilesystemencoding()``` works fine, while on Windows ```sys.getfilesystemencoding()``` returns ```'mbcs'```, which is not what virtualenv needs. If this comment is confusing I'll change or remove it.